spec/lib/whitehall_importer/create_edition_spec.rb

```diff
@@ -99,8 +99,14 @@
         whitehall_edition = build(
           :whitehall_export_edition,
           revision_history: [
-            build(:revision_history_event, whodunnit: 1, event: "create", state: "draft", created_at: 2.days.ago),
-            build(:revision_history_event, whodunnit: 2, event: "update", state: "published", created_at: 1.day.ago),
+            build(:revision_history_event, whodunnit: 1, event: "create", state: "draft", created_at: 5.days.ago),
+            build(:revision_history_event, whodunnit: 2, event: "update", state: "published", created_at: 4.days.ago),
+          ],
+          editorial_remarks: [
+            build(:note_event, author_id: 1, body: "Another note", state: "published", created_at: 2.days.ago),
+          ],
+          fact_check_requests: [
+            build(:fact_check_event, requestor_id: 2, email_address: "someone@somewhere.com", comments: "Hello World", state: "published", created_at: 3.days.ago, updated_at: 1.day.ago),
           ],
```

Comment:
Instead of passing static values here, I think we could assign each of these events to a variable, pass them to the edition and refer to them in the expect statements.  I don't think we care too much about what the values are, we just care that they're assigned to the correct attributes in the timeline entry. For example:
```
editiorial_remark = build(:note_event)
whitehall_edition = build(
... #some code
editorial_remarks: [editorial remark],
# rest of the code
```
`expect(edition.timeline_entries.third.created_at).to eq(editorial_remark.created_at)`

Reply:
Have updated as requested.